libs/vkd3d/command.c

```diff
@@ -11267,9 +11267,19 @@ static void d3d12_command_list_set_push_descriptor_info(struct d3d12_command_lis
                     : vk_info->device_limits.maxStorageBufferRange;
 
             resource = vkd3d_va_map_deref(&list->device->memory_allocator.va_map, gpu_address);
-            descriptor->info.buffer.buffer = resource->vk_buffer;
-            descriptor->info.buffer.offset = gpu_address - resource->va;
-            descriptor->info.buffer.range = min(resource->size - descriptor->info.buffer.offset, max_range);
+
+            if (resource)
+            {
+                descriptor->info.buffer.buffer = resource->vk_buffer;
+                descriptor->info.buffer.offset = gpu_address - resource->va;
+                descriptor->info.buffer.range = min(resource->size - descriptor->info.buffer.offset, max_range);
+            }
+            else
+            {
+                descriptor->info.buffer.buffer = VK_NULL_HANDLE;
```

Comment:
is there a reason why everything else has an `ERR` but this doesn't?

Reply:
Since this is called in response to hoisting and not direct application call, it felt more appropriate.